### PR TITLE
Add title to comparison section in the home page and improve its graphic

### PR DESCRIPTION
**Pull request by conradolandia**

### CODE REVIEW

**Comment by ccordoba12** [comment]
Thanks for the quick response Andrés! I only have a small comment: do you think we should add the v-shaped separator between the hero and this new section?

**Comment by conradolandia** [comment]
I added an optional divider to the hero section. Let me know what you think.

**Comment by ccordoba12** [comment]
Looks really good to me, thanks @conradolandia! I think the page looks more consistent that way, which is nice.

However, the name for that section is not that good, so let's wait to hear from @CAM-Gerlach about it.

File: src/routes/+page.js [hunk 1]
```diff
@@ -51,6 +51,8 @@ frontPage = {
         background: "/assets/media/bg.svg",
         columns: false,
         divider: true,
+        boxed: true,
+        title: "Comparison",
```
Comment (ccordoba12) [inline src/routes/+page.js]:
```suggestion
        title: "The Spyder advantage",
```

This is the title suggest by @CAM-Gerlach for this page and I think it's really good, so let's use it instead.

**Review by ccordoba12 — APPROVED** [review]
Looks good to me now, thanks @conradolandia!

**Pull request merged**